liferay-echo/liferay-portal
# LPS-180588 Use clay TreeView to render content page elements tree part I

Motivation
=======
What is the problem this work solves, including
[Link to task](https://issues.liferay.com/browse/LPS-151678)
[Link to subtask](https://issues.liferay.com/browse/LPS-180588)

Proposed Solution
========
Use Clay TreeView for rendering the tree nodes and use `actions` property of `ClayTreeView.Item` for rendering the actions available in each node. 

Steps to Verify:
----------------
1. Set feature.flag.LPS-151678=true
1. Go to home page and click on edit
1. In the left sidebar click on Browser section
2. Open the tree nodes and check that all the actions work for each node (rename, hide/show,  save composition duplicate, delete)
1. Set feature.flag.LPS-151678=true, restart the server and check the everything works as before


Screenshots:
-----------------------

![Captura de pantalla 2023-04-03 a las 13 59 09](https://user-images.githubusercontent.com/3276218/229502777-8b9f0dfc-541c-4a28-861b-4d67f6ed7664.png)
